src/js/view/controls/components/timeslider.js

```diff
@@ -69,12 +69,14 @@ function reasonInteraction() {
 }
 
 class TimeSlider extends Slider {
-    constructor(_model, _api) {
+    constructor(_model, _api, _accessibilityContainer) {
         super('jw-slider-time', 'horizontal');
 
         this._model = _model;
         this._api = _api;
 
+        this.timeUpdateKeeper = _accessibilityContainer.querySelector('.jw-time-update');
```

Comment:
Pass the `.jw-time-update` element to the constructor rather than the entire `_accessibilityContainer`.